lib/tsconfig-node-esm.json

```diff
@@ -1,9 +1,8 @@
 {
-  "extends": "./tsconfig-common.json",
+  "extends": "./tsconfig.json",
   "compilerOptions": {
-    "module": "ES2020",
+    "module": "es2020",
```

Comment:
sonarcloud is recommending this to be lowercase?  after doing a scan, the internet can't decide if it should be upper or lower.

Reply:
I went with lower for consistency with the typescript docs: https://www.typescriptlang.org/tsconfig#module